test/test-fs-event.c

```diff
@@ -389,8 +401,10 @@ TEST_IMPL(fs_event_watch_dir_recursive) {
 
   uv_run(loop, UV_RUN_DEFAULT);
 
-  ASSERT(fs_event_cb_called == fs_event_created + fs_event_removed);
-  ASSERT(close_cb_called == 2);
+  /* We do not need to check the number of fs_event_cb_called invocations
+   * because for the loop to terminate, we already know the number of create,
+   * remove and total fs_event_cb calls are correct. */
```

Comment:
I like me some asserts please :-) (what you just posted sounds reasonable, perhaps the comment should mention that on certain platforms you could get mor events and yada yada yada)